permission_handler_platform_interface/lib/src/permissions.dart

```diff
@@ -147,45 +152,45 @@ class Permission {
   /// https://support.google.com/googleplay/android-developer/answer/9214102#zippy=
   static const manageExternalStorage = Permission._(22);
 
-  ///Android: Allows an app to create windows shown on top of all other apps
-  ///iOS: Nothing
+  /// Android: Allows an app to create windows shown on top of all other apps
+  /// iOS: Nothing
   static const systemAlertWindow = Permission._(23);
 
-  ///Android: Allows an app to request installing packages.
-  ///iOS: Nothing
+  /// Android: Allows an app to request installing packages.
+  /// iOS: Nothing
   static const requestInstallPackages = Permission._(24);
 
-  ///Android: Nothing
-  ///iOS: Allows user to accept that your app collects data about end users and
-  ///shares it with other companies for purposes of tracking across apps and
-  ///websites.
+  /// Android: Nothing
+  /// iOS: Allows user to accept that your app collects data about end users and
+  ///   shares it with other companies for purposes of tracking across apps and
+  ///   websites.
```

Comment:
nit: redundant spacing

```suggestion
  /// shares it with other companies for purposes of tracking across apps and
  /// websites.
```